packages/core/src/App/Containers/RealAccountSignup/personal-details.jsx

```diff
@@ -211,11 +193,95 @@ class PersonalDetails extends React.Component {
+                                                                />
+                                                            )}
+                                                        </Field>
+                                                    )}
+                                                    {'tax_identification_number' in this.props.value && (
+                                                        <FormInputField
+                                                            name='tax_identification_number'
+                                                            label={localize('Tax identification number')}
+                                                            placeholder={localize('Tax identification number')}
+                                                        />
+                                                    )}
+                                                    {'tax_identification_confirm' in this.props.value && (
+                                                        <Field name='tax_identification_confirm'>
+                                                            {field => (
+                                                                <Checkbox
+                                                                    {...field}
+                                                                    data-lpignore
+                                                                    onChange={() =>
+                                                                        setFieldValue(
+                                                                            field.name,
+                                                                            !values[field.name],
+                                                                            true
+                                                                        )
+                                                                    }
+                                                                    value={values[field.name]}
+                                                                    label={localize(
+                                                                        'I hereby confirm that the tax information i provided is true and complete.I will also inform Binary Investments ( Europe ) Ltd. about any changes to this information.'
+                                                                    )}
+                                                                />
+                                                            )}
+                                                        </Field>
+                                                    )}
+                                                </>
+                                            )}
+                                            {/* TODO: [deriv-eu] Remove account opening reason once api is optional */}
```

Comment:
Can we move this comment to a `//` comment on line 256 instead?
These template comments end up on the DOM somehow.